### DISCUSSION TITLE
Question about modifying infoview window

### DISCUSSION THREAD

**Discussion by zaxioms0** [Julian/lean.nvim#247] [General]
Just kind of a random thought, I think would be really cool to add some things to the infoview window (in particular, adding custom symbols, fonts, etc) does anyone know how I could go about that?

**Comment by Julian** [comment]
Hey! Maybe you're looking for / interested in #172 -- which would allow adding (or removing) data from the infoview's display from your config? We'll get to it at some point.

But would love to hear more about what sorts of modifications you're thinking of, if you can share a bit more about the idea.

> adding custom symbols, fonts

For this, just to make sure I'm not misunderstanding, your terminal controls fonts, and you can't really change them inside of a program running in it (but yeah let me know if I misunderstood this part!)

**Reply by zaxioms0** [reply]
I think #172 might be what I was looking for! (fonts was a bad example, I more so meant custom symbols and information haha) In particular, I want to replace some of the information like "goals accomplished" with some custom text or change how the types of variables are displayed with custom highlighting and whatnot

**Comment by Julian** [comment]
Aha, right, makes sense, yep that's precisely what #172 will try to do -- any thoughts on design or functionality are of course welcome.
